pkg/occlient/occlient_test.go

```diff
@@ -6,10 +6,67 @@ import (
 	"os"
 	"reflect"
 	"testing"
+
+	fkrouteclientset "github.com/openshift/client-go/route/clientset/versioned/fake"
 )
 
-func TestGetOcBinary(t *testing.T) {
+func TestCreateRoute(t *testing.T) {
 
+	fkrouteClient := fkrouteclientset.NewSimpleClientset().Route()
+	fkclient := &Client{routeClient: fkrouteClient}
+
+	tests := []struct {
+		name    string
+		service string
+		labels  map[string]string
+		//		want    string
+		wantErr bool
+	}{
+		{
+			name:    "Case : mailserver",
+			service: "mailserver",
+			labels: map[string]string{
+				"SLA": "High",
+				"app.kubernetes.io/component-name": "backend",
+				"app.kubernetes.io/component-type": "python",
+			},
+			//			want:
+			wantErr: false,
+		},
+
+		{
+			name:    "Case : blog",
+			service: "blog",
+			labels: map[string]string{
+				"SLA": "High",
+				"app.kubernetes.io/component-name": "backend",
+				"app.kubernetes.io/component-type": "golang",
+			},
+			//			want:
+			wantErr: false,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			_, err := fkclient.CreateRoute(tt.service, tt.labels)
```

Comment:
LGTM!

Reply:
A few things need to be cleaned up to make it look neat, but since it is a WIP, I am not going to add those comments right away. or should I?